tilauskasittely/otsik.inc

```diff
@@ -3739,7 +3739,12 @@ if (isset($jatka)) {
       }
     }
 
-    if (!isset($eilahe) and $srow['eilahetetta'] != '' or (!isset($srow['eilahetetta']) and $yhtiorow["kerataanko_valmistukset"] == "E" and $toim == "VALMISTAASIAKKAALLE") or (!isset($srow['eilahetetta']) and $yhtiorow['lahetteen_tulostustapa'] == 'I') or (!isset($srow['eilahetetta']) and $toimtapa_apuro['nouto'] != '' and $yhtiorow['nouto_suoraan_laskutukseen'] == 'K')) $eilahe = 'o';
+    if (!isset($eilahe) and $srow['eilahetetta'] != '' or 
+      (!isset($srow['eilahetetta']) and $yhtiorow["kerataanko_valmistukset"] == "E" and $toim == "VALMISTAASIAKKAALLE") 
+      or (!isset($srow['eilahetetta']) and $yhtiorow['lahetteen_tulostustapa'] == 'I') 
+      or (!isset($srow['eilahetetta']) and $toimtapa_apuro['nouto'] != '' and $yhtiorow['nouto_suoraan_laskutukseen'] == 'K')
+      or $kukarow["eilahetetta"] == 'o') 
```

Comment:
@tonyselkamo @satuaalto Käyttäjän takana ominaisuus on nimetty näin "Tilaukset oletuksena suoraan laskutukseen". Tällä koodilla homma ei toimi enää noin. Näin koodattuna homma toimii pikatailauksella nyt näin: "Tilaukset aina suoraan laskutukseen". 

Pikatilauksen oletukset setataan luo_myyntitilausotsikko()-funkakrissa. Tsekatkaa, että settaus tehdään siellä saman, tai vastaavan, kaavan mukaan kuin otsik.inc:ssä.